spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/StreamCommands.java

```diff
@@ -150,11 +154,19 @@ public String deployStream(
 			propertiesToUse = DeploymentPropertiesUtils.convert(props);
 			break;
 		case -1: // Neither option specified
-			propertiesToUse = Collections.<String, String>emptyMap();
+			if(useSkipper) {
+				propertiesToUse = new HashMap<>(1);
+			}
+			else {
+				propertiesToUse = Collections.<String, String>emptyMap();
+			}
 			break;
 		default:
 			throw new AssertionError();
 		}
+		if(useSkipper) {
```

Comment:
While this helps to send the `actual` property as the deployment property, we could still keep the boolean flag instead of the actual property which comes from the `data flow server core`.